From d8b8ef7ecb5a22a28bc2af71bd2b5830c1b072f6 Mon Sep 17 00:00:00 2001
From: Paul Yim <paul.j.yim@gmail.com>
Date: Fri, 11 Dec 2020 17:40:16 -0500
Subject: [PATCH] load library from library bin on Windows

---
 rtree/core.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/rtree/core.py b/rtree/core.py
index 2549090..a9421e6 100644
--- a/rtree/core.py
+++ b/rtree/core.py
@@ -121,7 +121,8 @@ if os.name == 'nt':
         lib_path = os.path.join(sys.prefix, "Library", "bin")
         rt = _load_library(lib_name, ctypes.cdll.LoadLibrary, (lib_path,))
     else:
-        rt = _load_library(lib_name, ctypes.cdll.LoadLibrary)
+        lib_path = os.path.join(sys.prefix, "Library", "bin")
+        rt = _load_library(lib_name, ctypes.cdll.LoadLibrary, (lib_path,))
     if not rt:
         raise OSError("could not find or load %s" % lib_name)
 
-- 
2.20.1 (Apple Git-117)

